### PR TITLE
[AMORO-1528] Add metrics document

### PR DESCRIPTION
**Pull request by zhoujinsong**
## Why are the changes needed?

Add a document for metrics.

## Brief change log

- Move metric description in Dashboard to using-table page
- Add metric document

## How was this patch tested?

- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] Run test locally before making a pull request

## Documentation

- Does this pull request introduce a new feature? (no)
- If yes, how is the feature documented? (not applicable)

### CODE REVIEW

File: docs/user-guides/using-tables.md [hunk 1]
```diff
@@ -216,3 +216,37 @@ DROP TABLE test_db.test_log_store;
...
+- **Optimizing Status**: The current optimizing status of the table, including idle, pending, planning, minor, major, full, committing.
+  - idle: means that self-optimizing is not required on the table.
+  - pending: means that self-optimizing is required on the table and is waiting for resources.
+  - planning: means that self-optimizing process is being planed.
+  - minor: means that minor optimizing is being executed on the table. 
+  - major: means that major optimizing is being executed on the table.
+  - full: means that full optimizing is being executed on the table.
+  - committing: means that self-optimizing process is being committed.
+- **Duration**: The duration of the current status.
+- **File Count**: The total number of files involved in the current Self-optimizing, including base, insert, eq-delete, and pos-delete file types.
```
Comment (XBaith) [inline docs/user-guides/using-tables.md:249]:
```suggestion
- **File Count**: The total number of files involved in the current self-optimizing, including base, insert, eq-delete, and pos-delete file types.
```

File: docs/user-guides/using-tables.md [hunk 2]
```diff
@@ -216,3 +216,37 @@ DROP TABLE test_db.test_log_store;
...
+- **File Size**: The total size of files involved in the current Self-optimizing.
```
Comment (XBaith) [inline docs/user-guides/using-tables.md:250]:
```suggestion
- **File Size**: The total size of files involved in the current self-optimizing.
```

File: docs/user-guides/using-tables.md (hunk 1 above)
Comment (XBaith) [reply docs/user-guides/using-tables.md:249]: Should we describe it in more detail? For example with a self-optimising process?

**Review by XBaith — COMMENTED** [review]
I drop some comments, PTAL

File: docs/user-guides/using-tables.md (hunk 1 above)
Comment (zhoujinsong) [reply docs/user-guides/using-tables.md:249]: Do you mean that we can add some description about the optimizing process tab in the table detail page here?

If yes, maybe we can add it in another PR.

**Review by XBaith — APPROVED** [review]
LGTM
